tgui/packages/tgui/interfaces/AirlockController.tsx

```diff
@@ -0,0 +1,168 @@
+import { useBackend } from '../backend';
+import { Box, Button, Icon, LabeledList, Section } from '../components';
+import { Window } from '../layouts';
+
+type AirlockControllerData = {
+  airlockState: string;
+  sensorPressure: number;
+  pumpStatus: string;
+  interiorStatus: string;
+  exteriorStatus: string;
+};
+
+type AirlockStatus = {
+  primary: string;
+  icon?: string;
+  color?: string;
+};
+
+const defaultStatus: AirlockStatus = {
+  primary: 'Unknown',
+  icon: '',
+  color: 'average',
+};
+
+export const AirlockController = (_, context) => {
+  const { data } = useBackend<AirlockControllerData>(context);
+  const { airlockState, pumpStatus, interiorStatus, exteriorStatus } = data;
+  let currentStatus: AirlockStatus = getAirlockStatus(airlockState);
```

Comment:
Почему `let`? Нигде не изменяется же
```suggestion
  const currentStatus: AirlockStatus = getAirlockStatus(airlockState);
```